src/tiler/server.js

```diff
@@ -55,12 +55,22 @@ var interactivity = {
         tableName = req.params.table;
         stream_order = 0;  // All streams
 
-        if (zoom <= 5) {
-            stream_order = 7;
-        } else if (zoom <= 6) {
-            stream_order = 6;
-        } else if (zoom <= 8) {
-            stream_order = 5;
+        if (tableName === tables.drb_streams_v2) {
+            // drb_zoom_levels: { zoomLevel : stream_order }
+            drb_zoom_levels = {
+                1:7, 2:7, 3:7, 4:7, 5:7, 6:6, 7:6, 8:5, 9:5, 10:4,
+                11:3, 12:2, 13:0, 14:0, 15:0, 16:0, 17:0, 18: 0
```

Comment:
I checked this again and I'm no longer getting the 400 error from the tiler on zooming in all the way for the default basemap. 

When I switch basemaps and zoom in further, I don't see a the 400 error but I also don't see any streams. I checked on staging and it looks like this is how it behaves there, too (maybe the tiler doesn't go past 18 for this overlay?). Just wanted to make sure this is the desired behavior?